contrib/murmurhash/src/MurmurHash3.cpp

```diff
@@ -54,9 +54,7 @@ inline uint64_t rotl64 ( uint64_t x, int8_t r )
 
 FORCE_INLINE uint32_t getblock32 ( const uint32_t * p, int i )
 {
-  uint32_t res;
-  memcpy(&res, p + i, sizeof(res));
-  return res;
+  return p[i];
```

Comment:
Looks like ub patch to me.
@alexey-milovidov, could you confirm?

Reply:
Yes.

Reply:
I have restored this fix.